plugin/storage/es/factory.go

```diff
@@ -59,19 +59,25 @@ type Factory struct {
 
 	newClientFn func(c *config.Configuration, logger *zap.Logger, metricsFactory metrics.Factory) (es.Client, error)
 
-	primaryConfig *config.Configuration
-	archiveConfig *config.Configuration
+	config *config.Configuration
 
-	primaryClient atomic.Pointer[es.Client]
-	archiveClient atomic.Pointer[es.Client]
+	client atomic.Pointer[es.Client]
 
 	watchers []*fswatcher.FSWatcher
```

Comment:
does this still need to be a list, or just exactly one watcher? If one, and it's for a password, I would rename it to pwdFileWatcher

Reply:
@yurishkuro done